net/atftp/files/atftpd.init

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2020 OpenWrt.org
+
+START=95
+PIDFILE=/tmp/run/atftpd.pid
+
+start() {
+	config_load atftpd
+	config_get SRV service path "/srv/tftp"
+	config_get PORT service port 69
+
+        atftpd --pidfile $PIDFILE --user root.root --port $PORT --daemon $SRV
```

Comment:
whitespace difference?

Reply:
Fixed PKG_RELEASE. Fixed whitespace.

I don't understand the CI testing. There are two packages generated, how is that handled?  atftp --version && atftpd --version, perhaps?